newfile.php

```diff
@@ -1,5 +1,6 @@
 <?php
 echo  "hello world";
+echo  "PR test"
```

Comment:
Add information for PR
```suggestion
echo  "PR test on GitHub"
```